libraries/chain/transaction_context.cpp

```diff
@@ -134,8 +134,7 @@ namespace eosio { namespace chain {
       rl.update_account_usage( bill_to_accounts, block_timestamp_type(control.pending_block_time()).slot );
 
       // Calculate the highest network usage and CPU time that all of the billed accounts can afford to be billed
-      int64_t account_net_limit = 0;
-      int64_t account_cpu_limit = 0;
+      int64_t account_net_limit = 0,account_cpu_limit = 0;
```

Comment:
why? This just makes it less readable.

Reply:
"Why?" - m just trying to contribute only what is correct, doesn't deteriorate and if spotted to enhance. It is also a matter of style and saving code storage space as I have explained already before. It's up to you to accept the amendment(restyling or similar in this specific case to save code storage rows) or leave it as per your view. Let us agree on this.